### PR TITLE
Update troposphere library

### PR DESCRIPTION
**Pull request by zaro0508**
When attempting to deploy troposphere template I get the following error

```
AttributeError: 'Template' object has no attribute 'to_yaml'

```

Accoring to https://github.com/cloudtools/troposphere/issues/759  it
seems like we need to update trophoshere to a version where the
`to_yaml` was added back.  The current dependency to trophosphere (v2.0.2)
is pretty old anyways so we are updating the version to the latest 2.x
release

The latest version is 3.0.3 however updating to 3.x causes sceptre unit
tests to fail so we are sticking to the latest 2.x version.

### CODE REVIEW

**Review by craighurley — APPROVED** [review]

**Review by tarkatronic — COMMENTED** [review]
Given that we know 2.0.x doesn't work, should we keep the restriction higher? Perhaps `>=2.1.0,<3`? Or even `>=2.2.0,<3`?

It might be worth figuring out what versions actually work. Because with this specification, somebody could still install a broken version.

**Comment by zaro0508** [comment]
> It might be worth figuring out what versions actually work. Because with this specification, somebody could still install a broken version.

Good point @tarkatronic , i have verified that the latest 2.x  version (v2.7.1) works so i have adjusted the version accordingly.

File: requirements/prod.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@ PyYaml>=5.1,<6.0
 sceptre-cmd-resolver>=1.1.3,<2
 sceptre-file-resolver>=1.0.4,<2
 six>=1.11.0,<2.0.0
-troposphere>=2.0.0,<2.1.0
+troposphere>=2.7,<3
```
Comment (craighurley) [inline requirements/prod.txt:12]: @zaro0508 recently in slack, you mentioned `2.0.2` works.  Should this be `>=2.0.2,<3` or are there versions greater than `2.0.2` and less than `2.7` that cause issues?

Comment (zaro0508) [reply requirements/prod.txt:12]: sorry, i think the slack conversation was a bit confusing.  To clarify, I get the `AttributeError` on a troposphere template deployment when run with ver 2.0.2.  I reran the deployment with troposphere ver 2.7.1 and it works without error.  I did not test with any other versions.

Comment (craighurley) [reply requirements/prod.txt:12]: got it

**Review by craighurley — APPROVED** [review]

**Comment by zaro0508** [comment]
are you satisfied @tarkatronic ?

**Review by tarkatronic — APPROVED** [review]
Much better!

**Pull request merged**